src/devices/Hx711/HX711CalibrationNotDoneException.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.HX711
+{
+    /// <summary>
+    /// Exception thorw if Hx711 miss calibration process
+    /// </summary>
+    public class HX711CalibrationNotDoneException : Exception
+    {
+        private const string MESSAGE = "HX711 component need a calibration process first.";

```

Comment:
we're not using uppercase for constants, so, it should just be Message

Reply:
and also, why creating a specific exception?

Reply:
I thought about creating a specific exception because I thought it cleared up the problem. Better to launch a generic one?